plugins/scaleway/api_key.go

```diff
@@ -0,0 +1,127 @@
+			{
+				Name:                fieldname.AccessKeyID,
+				MarkdownDescription: "The ID of the API Key used to authenticate to Scaleway.",
+				Secret:              false,
+				Composition: &schema.ValueComposition{
+					Length: 20,
+					Prefix: "SCW",
+					Charset: schema.Charset{
+						Uppercase: true,
+						Digits:    true,
+					},
+				},
+			},
+			{
+				Name:                fieldname.SecretAccessKey,
+				MarkdownDescription: "The secret access key used to authenticate to Scaleway.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 36,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+						Specific:  []rune{'-'},
+					},
+				},
+			},
+			{
+				Name:                fieldname.DefaultOrganization,
+				MarkdownDescription: "The default organization ID to use for this access key.",
+				Secret:              false,
+				Composition: &schema.ValueComposition{
+					Length: 36,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+						Specific:  []rune{'-'},
+					},
+				},
+			},
+			{
+				Name:                fieldname.DefaultRegion,
+				MarkdownDescription: "The default region to use for this access key.",
+				Optional:            true,
+			},
+			{
+				Name:                fieldname.DefaultZone,
+				MarkdownDescription: "The default zone to use for this access key.",
+				Optional:            true,
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer: importer.TryAll(
+			importer.TryEnvVarPair(defaultEnvVarMapping),
+			TryScalewayConfigFile(),
+		)}
+}
+
+var defaultEnvVarMapping = map[string]sdk.FieldName{
+	"SCW_ACCESS_KEY":              fieldname.AccessKeyID,
+	"SCW_SECRET_KEY":              fieldname.SecretAccessKey,
+	"SCW_DEFAULT_ORGANIZATION_ID": fieldname.DefaultOrganization,
+	"SCW_DEFAULT_REGION":          fieldname.DefaultRegion,
+	"SCW_DEFAULT_ZONE":            fieldname.DefaultZone,
+}
+
+func TryScalewayConfigFile() sdk.Importer {
+	file := os.Getenv("SCW_CONFIG_PATH")
+	if file == "" {
+		file = "~/.config/scw/config.yaml"
+	}
+	return importer.TryFile(file, func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
+		var config Config
+		if err := contents.ToYAML(&config); err != nil {
+			out.AddError(err)
+			return
+		}
+
+		if config.AccessKey == "" || config.SecretKey == "" {
+			return
+		}
+
+		fields := make(map[sdk.FieldName]string)
+		fields[fieldname.AccessKeyID] = config.AccessKey
+		fields[fieldname.SecretAccessKey] = config.SecretKey
+		if config.DefaultOrganizationID != "" {
+			fields[fieldname.DefaultOrganization] = config.DefaultOrganizationID
+		}
+		if config.DefaultRegion != "" {
+			fields[fieldname.DefaultRegion] = config.DefaultRegion
+		}
+		if config.DefaultZone != "" {
+			fields[fieldname.DefaultZone] = config.DefaultZone
+		}
+		out.AddCandidate(sdk.ImportCandidate{
```

Comment:
It seems that Scaleway config file supports the concepts of profiles. Could we import the profile name as a [Namehint](https://pkg.go.dev/github.com/1Password/shell-plugins/sdk#ImportCandidate)? You'll need to set up at least two profiles to test the importer. Example:

```
profiles:
  first:
    access_key: redacted
    secret_key: redacted
    default_organization_id: redacted
    default_project_id: redacted
    default_zone: fr-par-1
    default_region: fr-par
    # api_url: https://api.scaleway.com
    # insecure: false

  second:
    access_key: redacted
    secret_key: redacted
    default_organization_id: redacted
    default_project_id: redacted
    default_zone: fr-par-1
    default_region: fr-par
    # api_url: https://api.scaleway.com
    # insecure: false
```

There's an example of how to import multiple profiles in the Fastly shell plugin.